Objective-C/CBLBlob.mm

```diff
@@ -42,19 +42,20 @@
 // Stack buffer size when reading NSInputStream
 static const size_t kReadBufferSize = 8*1024;
 
-static NSString* const kTypeMetaProperty = @kC4ObjectTypeProperty;
-static NSString* const kDigestMetaProperty = @kC4BlobDigestProperty;
-static NSString* const kDataMetaProperty = @kC4BlobDataProperty;
-static NSString* const kLengthMetaProperty = @"length";
-static NSString* const kContentTypeMetaProperty = @"content_type";
-static NSString* const kBlobType = @kC4ObjectType_Blob;
+NSString* const kCBLBlobType = @kC4ObjectType_Blob;
+NSString* const kCBLBlobTypeMetaProperty = @kC4ObjectTypeProperty;
+NSString* const kCBLBlobDigestMetaProperty = @kC4BlobDigestProperty;
```

Comment:
Let's do to match CBL-C.

**Blob:**
kCBLBlobType
kCBLBlobDigestProperty
kCBLBlobContentTypeProperty
kCBLBlobLengthProperty

**Type**
kCBLTypeProperty